README.md

```diff
@@ -1,38 +1,51 @@
 # Rosetta Kinase CM
+This repository is for our project <b><i>Comparative Modeling Pipeline of Protein Kinase Inhibitors</i></b>. 
 
-<!-- <figure class="image">
+
+<figure class="image">
   <img src="pipeline.png">
-  <figcaption>Figure: Overview of Rosetta Kinase CM pipeline</figcaption>
-</figure> -->
+  <figcaption>Figure: Overview of KinBADL pipeline</figcaption>
```

Comment:
@wgabrielong KinBADL is a temp name for our deep learning model. So here we can stick with the Rosetta Kinase CM.

Reply:
@pkiruba sorry, will make the change. I used the insert figure text from KinBADL but forgot to change the caption.